tests/compas/datastructures/test_mesh.py

```diff
@@ -459,7 +521,9 @@ def test_centroid():
     assert mesh.centroid() == [0.0, 0.0, 0.5]
 
     mesh = Mesh.from_obj(compas.get('quadmesh.obj'))
-    assert mesh.centroid() == [2.508081952064351, 2.554046390557884, 1.2687133268242006]
+    assert mesh.centroid() == [
```

Comment:
why is this split? max-line-length for compas flake8 linting is set to 180...

Reply:
Somethings up with my config, will fix.